packages/sindarian-ui/src/components/ui/calendar/index.tsx

```
@@ -222,7 +223,7 @@ function CalendarDayButton({
       role="gridcell"
       tabIndex={modifiers.focused ? 0 : -1}
       className={cn(
-        'data-[selected-single=true]:bg-primary data-[selected-single=true]:text-primary-foreground data-[range-middle=true]:bg-muted data-[range-middle=true]:text-foreground data-[range-start=true]:bg-primary data-[range-start=true]:text-primary-foreground data-[range-end=true]:bg-primary data-[range-end=true]:text-primary-foreground group-data-[focused=true]/day:border-ring group-data-[focused=true]/day:ring-ring/50 dark:hover:text-foreground relative isolate z-10 flex aspect-square size-auto w-full min-w-(--cell-size) flex-col gap-1 border-0 leading-none font-normal group-data-[focused=true]/day:relative group-data-[focused=true]/day:z-10 group-data-[focused=true]/day:ring-[3px] data-[range-end=true]:rounded-(--cell-radius) data-[range-end=true]:rounded-r-(--cell-radius) data-[range-middle=true]:rounded-none data-[range-start=true]:rounded-(--cell-radius) data-[range-start=true]:rounded-l-(--cell-radius) [&>span]:text-xs [&>span]:opacity-70',
+        'data-[today=true]:bg-accent data-[today=true]:text-accent-foreground data-[today=true]:font-bold data-[selected-single=true]:bg-accent data-[selected-single=true]:text-accent-foreground data-[range-middle=true]:bg-accent-mute data-[range-middle=true]:text-foreground data-[range-start=true]:bg-accent data-[range-start=true]:text-accent-foreground data-[range-end=true]:bg-accent data-[range-end=true]:text-accent-foreground focus-visible:border-accent focus-visible:ring-accent/50 relative isolate z-10 flex aspect-square size-auto w-full min-w-(--cell-size) flex-col gap-1 border-0 leading-none font-normal focus-visible:ring-[3px] focus-visible:outline-none data-[range-end=true]:rounded-(--cell-radius) data-[range-end=true]:rounded-r-(--cell-radius) data-[range-end=true]:font-bold data-[range-middle=true]:rounded-none data-[range-start=true]:rounded-(--cell-radius) data-[range-start=true]:rounded-l-(--cell-radius) data-[range-start=true]:font-bold data-[selected-single=true]:font-bold [&>span]:text-xs [&>span]:opacity-70',
```

ESLint

[error] Replace `today=true]:font-bold·data-[selected-single=true]:bg-accent·data-[selected-single=true]:text-accent-foreground·data-[range-middle=true]:bg-accent-mute·data-[range-middle=true]:text-foreground·data-[range-start=true]:bg-accent·data-[range-start=true]:text-accent-foreground·data-[range-end=true]:bg-accent·data-[range-end=true]:text-accent-foreground·focus-visible:border-accent·focus-visible:ring-accent/50·relative·isolate·z-10·flex·aspect-square·size-auto·w-full·min-w-(--cell-size)·flex-col·gap-1·border-0·leading-none·font-normal·focus-visible:ring-[3px]·focus-visible:outline-none·data-[range-end=true]:rounded-(--cell-radius)·data-[range-end=true]:rounded-r-(--cell-radius)·data-[range-end=true]:font-bold·data-[range-middle=true]:rounded-none·data-[range-start=true]:rounded-(--cell-radius)·data-[range-start=true]:rounded-l-(--cell-radius)·data-[range-start=true]:font-bold·data-[selected-single` with `selected-single=true]:bg-accent·data-[selected-single=true]:text-accent-foreground·data-[range-middle=true]:bg-accent-mute·data-[range-middle=true]:text-foreground·data-[range-start=true]:bg-accent·data-[range-start=true]:text-accent-foreground·data-[range-end=true]:bg-accent·data-[range-end=true]:text-accent-foreground·focus-visible:border-accent·focus-visible:ring-accent/50·relative·isolate·z-10·flex·aspect-square·size-auto·w-full·min-w-(--cell-size)·flex-col·gap-1·border-0·leading-none·font-normal·focus-visible:ring-[3px]·focus-visible:outline-none·data-[range-end=true]:rounded-(--cell-radius)·data-[range-end=true]:rounded-r-(--cell-radius)·data-[range-end=true]:font-bold·data-[range-middle=true]:rounded-none·data-[range-start=true]:rounded-(--cell-radius)·data-[range-start=true]:rounded-l-(--cell-radius)·data-[range-start=true]:font-bold·data-[selected-single=true]:font-bold·data-[today`

(prettier/prettier)